docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -951,10 +956,139 @@ Step 6: First Boot
    Replace ``YOUR_USERNAME`` with your desired username::
 
      username=YOUR_USERNAME
+     
+   Choose one of the following:
+     
+   - Unencrypted homedir or whole-disk encryption
+     
+       ::
+
+         zfs create rpool/home/$username
+         adduser $username
+       
+   - Encrypted homdir per user, automatic decryption on login
+     
+       ::
+       
+         zfs create rpool/home/$username -o encryption=on -o keyformat=passphrase -o keylocation=prompt -o canmount=noauto -o org.openzfs.systemd:ignore=on
+         zfs mount rpool/home/$username
+         adduser $username
+         
+       **Note**: Use the same strong password for ZFS encryption and user password. Please note: After a breach of your password changing the ZFS password does not restore protection.
+       
+       Tell PAM to unlock the dataset key on login::
+       
+         printf "auth [success=1 default=ignore] pam_succeed_if.so service = sudo quiet\n\
+                 auth optional pam_exec.so expose_authtok /usr/local/sbin/unlock-key-zfs-homedir\n"\
+           >> /etc/pam.d/common-auth
+           
+       The second line calls a new unlock-script, the first line disables it if we just sudo.
```

Comment:
Why do we want to disable it if we just sudo? I'm not saying you're wrong, but can you explain it a little more. To me in comments here is fine. The explanation doesn't have to be in the document itself at this point.

Reply:
Systemd doesn't start a user session when using sudo. There is no point in unlocking our dataset if no one is going to mount it.